### PR TITLE
Added an 'Edit Artifact' functionality. Added 'Register Collection' functionality

### PR DESCRIPTION
**Pull request by Itaho**
Reports come next

### CODE REVIEW

**Comment by Itaho** [comment]
There is a weird check that fails with Maven. I'm not sure what its meant for but its flagging the HTML edit to employee.html and i have no clue as to what it means. If its a problem then I'll figure it out but I didn't have any issues running it

File: src/main/java/com/mfahproj/webapp/App.java [hunk 1]
```diff
@@ -69,10 +69,15 @@ private static void setRoutes(HttpServer server, MiddlewareHandler.HttpRequestCa
         server.createContext("/member/register", new MiddlewareHandler(new RegisterMemberHandler(), callback));
         server.createContext("/employee/register", new MiddlewareHandler(new RegisterEmployeeHandler(), callback));
         server.createContext("/artifact/register", new MiddlewareHandler(new ArtifactHandler(), callback));
+        server.createContext("/collection/register", new MiddlewareHandler(new ArtifactHandler(), callback));
```
Comment (Ohkthx) [inline src/main/java/com/mfahproj/webapp/App.java]: The handler called for registering a collection is `ArtifactHandler()` here.

File: src/main/java/com/mfahproj/webapp/App.java [hunk 2]
```diff
@@ -69,10 +69,15 @@ private static void setRoutes(HttpServer server, MiddlewareHandler.HttpRequestCa
...
+        server.createContext("/program/register", new MiddlewareHandler(new ArtifactHandler(), callback));
```
Comment (Ohkthx) [inline src/main/java/com/mfahproj/webapp/App.java]: The handler called for registering a program is `ArtifactHandler()` here.

File: src/main/java/com/mfahproj/webapp/App.java [hunk 3]
```diff
@@ -69,10 +69,15 @@ private static void setRoutes(HttpServer server, MiddlewareHandler.HttpRequestCa
...
 
         // Used for editing content.
         server.createContext("/member/edit", new MiddlewareHandler(new EditMemberHandler(), callback));
         server.createContext("/employee/edit", new MiddlewareHandler(new EditEmployeeHandler(), callback));
+        server.createContext("/artifact/edit", new MiddlewareHandler(new EditArtifactHandler(), callback));
+        server.createContext("/collection/edit", new MiddlewareHandler(new EditArtifactHandler(), callback));
```
Comment (Ohkthx) [inline src/main/java/com/mfahproj/webapp/App.java]: The handler called for editing a collection is `EditArtifactHandler()` here.

File: src/main/java/com/mfahproj/webapp/App.java [hunk 4]
```diff
@@ -69,10 +69,15 @@ private static void setRoutes(HttpServer server, MiddlewareHandler.HttpRequestCa
...
+        server.createContext("/program/edit", new MiddlewareHandler(new EditArtifactHandler(), callback));
```
Comment (Ohkthx) [inline src/main/java/com/mfahproj/webapp/App.java]: The handler called for editing a program is `EditArtifactHandler()` here.

**Review by Ohkthx — CHANGES_REQUESTED** [review]
Handlers were not updated in App.java.

**Review by Itaho — COMMENTED** [review]
Fixed App.java and renamed ProgramHandler to RegisterProgramHandler

**Pull request closed without merging**